docs/mobile-apps/virtual-usb.md

```diff
@@ -15,164 +15,247 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
+  As a reminder, vUSB only works on private devices (marked with a <img src={useBaseUrl('img/virtual-usb/vusb-private.jpg')} alt="Sauce Labs Private Device Icon" width="15" />). A quick way to find your organization's private devices from the device list is to click **Filters** and toggle **Private Devices**.
 
-<img src={useBaseUrl('img/virtual-usb/vusb-privatedevices.jpg')} alt="Virtual USB Private Devices Filters" width="400" />
+  <img src={useBaseUrl('img/virtual-usb/vusb-privatedevices.jpg')} alt="Virtual USB Private Devices Filters" width="500" />
 
-5. Grab your Sauce Labs credentials (username and access key), which you can find under **Account** > **User settings**.
-6. Open a second command line terminal window, while keeping the first window running in the background. In the new window, run this command line to launch your vUSB session, authenticate your credentials (`username` and `accessKey`), and connect to your desired device:
+### Connect to Server
 
-```java
-java -jar vusb-client.jar startSession --username john.smith --accessKey ab015c1e-1997-4794-8g52-32f7fe110e03 --deviceName Motorola_Moto_Z_real
-```
-Your chosen device will be allocated specifically for your test session. It will not available for other users until your session is over.
+4. On your local machine, launch a command line terminal window and use `cd` to navigate to the folder where you downloaded the vUSB client.
+
+5. Enter the `server` command, followed by the `--datacenter US` (Sauce Labs U.S. Data Center) or `--datacenter EU` (Sauce Labs Europe Data Center). This establishes the connection from your local machine to our Real Device Cloud, where your private devices are hosted.
+  ```java
+  java -jar virtual-usb-client-2.0.0.jar server --datacenter US
+  ```
 
-7. When vUSB connects to your live testing session, it will return a success message along with a link you can click to watch the device running your test in real time. Here's an example of a success message:
+  **Set Up a Local Proxy (Optional)**
 
+  You can launch a proxy tunnel or device proxy tunnel using the proxy command options (run `java -jar virtual-usb-client-2.0.0.jar --help` and go to the `server` section).
+  <img src={useBaseUrl('img/virtual-usb/vusb-proxy.jpg')} alt="Virtual USB Proxy Command Options" width="500" />
+
+  **Set Environment Variables (Optional)**
+
+  You can set your Sauce Labs user credentials (`username` and `accessKey`) as environment variables. For more info, see [Using Environment Variables for Authentication Credentials](https://wiki.saucelabs.com/pages/viewpage.action?pageId=48365647#BestPracticesforRunningTests-UseEnvironmentVariablesforAuthenticationCredentials).
+
+### Start Test Session
+
+6. Choose one of the options below to establish the connection to your device and start your vUSB test session.
```

Comment:
We should be more clear about what this command is going to do, something like:

```
There are 2 ways to start a vUSB connection with a device. The first way is to start a new fresh vUSB session from the command line. The second way is to connect to an already existing manual live session in the Sauce Labs cloud.
We are going to start with a new fresh session...
```

Reply:
thanks, wim!  I must have missed the instructions for connecting to an existing Live session - can you share the steps?

I changed up the section a bit...


In this step, you'll establish the connection to your device and start your vUSB test session. There are two ways to do do this:

  **Option 1: Start a fresh new vUSB session from the command line**: Open a new command line terminal window, while keeping the first one with the server connection running in the background. Run the `startSession` command, followed by your `username` and `accessKey` for authentication, then the `--deviceName`:
<Tabs
  defaultValue="Android"
  values={[
    {label: 'Android', value: 'Android'},
    {label: 'iOS', value: 'iOS'},
  ]}>

  <TabItem value="Android">

  ```java
  java -jar virtual-usb-client-2.0.0.jar startSession --username john.smith --accessKey ab015c1e-xxxx-xxxx-xxxx-xxxxxxxxxxxx --deviceName Motorola_Moto_Z_real
  ```

  </TabItem>
  <TabItem value="iOS">

  ```java
  java -jar virtual-usb-client-2.0.0.jar startSession --username john.smith --accessKey ab015c1e-xxxx-xxxx-xxxx-xxxxxxxxxxxx --deviceName iPhone_XS
  ```

  </TabItem>
  </Tabs>

  If you'd like to use Sauce Connect Proxy, launch a tunnel in the Sauce Connect client, then add your `--tunnel-identifier` at the end of the above snippet. The vUSB client uses it to retrieve and secure test data.

   or

   **Option 2: Connect to an existing Live test session**: The second method is to connect to an existing manual live session you've got running in the Sauce Labs cloud.

Reply:
@kimsauce 

>  for connecting to an existing Live session - can you share the steps?

It was the previous code. so open a live testing session on the RDC and connect to the device with the device id with the `connect` command

Reply:
Oh I see, sorry i missed that. So something like this?

 **Option 2: Connect to an existing live testing session**: Launch a live test on Sauce Labs (**Live** > **Mobile App** > Select Your App > **Choose Device** > Find Your Private Device > **Launch**). Then, you'll need to connect the session to your vUSB client. Open a new command line terminal and use the `sessions` command plus your credentials.

   ```java
   java -jar virtual-usb-client.jar sessions --username john.smith --accessKey ab015c1e-xxxx-xxxx-xxxx-xxxxxxxxxxxx
   ```

   This will return a list of your open test sessions. Find the `sessionId` of your desired test, then run the `connect` command along with the `sessionId` and your credentials.

   ```java
   java -jar virtual-usb-client.jar connect ---username john.smith --accessKey ab015c1e-xxxx-xxxx-xxxx-xxxxxxxxxxxx --sessionId 1234567890
   ```

Reply:
@kimsauce 
Yes, this is good, but is there a reason why you've chooses this order

`Sauce Labs (Live > Mobile App > Select Your App > Choose Device > Find Your Private Device > Launch)`

I would use `Sauce Labs (Live > Cross Browser > Choose Device > Find Your Private Device > Launch)` This will start in the "most" clean state. They can push apps later through their IDE to debug them. You can't debug an app that is installed by Sauce

Reply:
I see! all done